<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="spinBox_1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>47</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>47</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_dinheiro">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dinheiro (em milh√µes)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pessoas">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pessoas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_risco">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>10</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QLabel" name="label_resposta">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>150</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ok</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
